erpnext/healthcare/doctype/patient_encounter/patient_encounter.json

```diff
@@ -345,7 +355,7 @@
  "show_name_in_global_search": 1,
  "sort_field": "modified",
  "sort_order": "DESC",
- "title_field": "patient",
+ "title_field": "patient_name",
```

Comment:
I think the Patient field should be the title field to identify the Patient uniquely. Also, the Patient Name field should be removed from standard filters as we can always search using Patient Name in Link fields. The Company field should be in the first section.